test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1059,6 +1059,14 @@ public static final class Builder {
 
         private Position position;
 
+        /**
+         * A private constructor for the builder,
+         * it should not be created directly.
+         * Use {@code PassFailJFrame.builder()} method instead.
+         */
+        private Builder() {
+        }
+
         public Builder title(String title) {
```

Comment:
So what about this one ? Are too many tests using it ?

Reply:
What about them? All the methods in `Builder` return `Builder`, that's perfectly fine.

The goal of this PR is to make impossible to write

```java
new PassFailJFrame.Builder();
```

which is replaced with

```java
PassFailJFrame.builder();
```

In the former case, the test explicitly depends on the fact that the builder pattern implementation is in the `PassFailJFrame.Builder` class.

In the latter case, I can change the `builder()` method to return another class. As long as that other class has the same set methods, like `instructions` and title`, which return the same object, nothing breaks. Encapsulation.

With this change, all the existing tests continue to run. Those tests which failed to compile because of a breaking API change are updated in this PR.

Reply:
Never mind. I wasn't reading it properly.